follow_fork/README.md

```diff
@@ -0,0 +1,27 @@
+Follow fork()
```

Comment:
Backticks around `fork`, both here and in the rest of the file.

Reply:
done